### PR TITLE
Added how Vim can use yamllint without any plugins

### PR DESCRIPTION
**Pull request by coderonline**

### CODE REVIEW

**Comment by polyzen** [comment]
Please remove the excess blank lines.

File: docs/text_editors.rst [hunk 1]
```diff
@@ -20,6 +27,7 @@ plugin, add this to your ``.vimrc``:
 
  let g:syntastic_yaml_checkers = ['yamllint']
 
+
```
Comment (polyzen) [inline docs/text_editors.rst]: This one too. :)

File: docs/text_editors.rst [hunk 2]
```diff
@@ -9,6 +9,13 @@ text editor.
 Vim
 ---
 
+In its standard installation the following command can be used to populate
+Vims location list with yamllints results:
```
Comment (adrienverge) [inline docs/text_editors.rst]:
```suggestion
Vim location list with yamllint results:
```
?

**Comment by coderonline** [comment]
I am still wondering how sqashing works with the web interface of gitlab and I am currently reading the manual.

**Comment by adrienverge** [comment]
This is GitHub, not GitLab, and I don't think you can squash with the web UI.

Just `git rebase -i origin/master` 😉
http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by polyzen** [comment]
@adrienverge, see "squash and merge": https://help.github.com/articles/about-pull-request-merges/

**Comment by adrienverge** [comment]
@polyzen yeah I know this feature. I prefer authors to follow this project's standard and write commit messages themselves.

**Comment by adrienverge** [comment]
@coderonline thanks for squashing!

- Again, it would be better if you used the commit message conventions (see https://github.com/adrienverge/yamllint/commits/master), something like "docs: Document usage in Vim without any plugins". It eases changelogging and review of Git history.
- You clicked "resolve" on my remark in the code, but didn't change anything. It seems like there are two extra "s" characters, aren't they?

**Comment by adrienverge** [comment]
Closing due to lack of response from the author.

**Pull request closed without merging**